datumaro/components/annotation.py

```diff
@@ -673,7 +680,7 @@ def __eq__(self, other):
         self_polygon = sg.Polygon(self.get_points())
         other_polygon = sg.Polygon(other.get_points())
         inter_area = self_polygon.intersection(other_polygon).area
-        return abs(self_polygon.area - inter_area) == 0.0
+        return abs(self_polygon.area - inter_area) < 1e-7
```

Comment:
Using `sys.float_info.epsilon` may be better than the hard-coded value 1e-7.

Reply:
`sys.float_info.epsilone` is `2.220446049250313e-16`. However, it's too tight so that there is a case that two polygons are same but the area difference can be greather than that.

Reply:
Instead of using it, I give a name to the number, `CHECK_POLYGON_EQ_EPSILONE = 1e-7`.

Reply:
cool